.github/actions/push-registry-app-files-supabase/src/index.js

```diff
@@ -0,0 +1,91 @@
+const fs = require("fs")
+const core = require("@actions/core");
+const { execSync } = require("child_process");
+
+import { createClient } from "@supabase/supabase-js";
+
+const supabaseUrl = "https://ywxbxfcsskoyjwnlpqms.supabase.co"
```

Comment:
Let's actually move this to env, I added a `SUPABASE_URI` var that holds this full URL with protocol